ruby/templates/api_client_typhoeus_partial.mustache

```diff
@@ -13,7 +13,7 @@
       sleep_time = 0.05
```

Comment:
can you update this comment to `# retry 500s and network errors` ?

Reply:
@svix-lucho updated, thanks!